### ISSUE TITLE
Topcoder tab icon (favicon) not showing

### ISSUE THREAD

**Issue by Chapre** [topcoder-platform/Training-Challenges#206]
**Steps to Reproduce:**
1. Go to any submitted challenge review page
2. look at the active tab icon (favicon) at the top, it looks like blank file icon

**Actual Result:**
![image](https://user-images.githubusercontent.com/13720708/121763090-37817880-cb31-11eb-88dd-6299df2ebd90.png)

**Expected Result:**
![image](https://user-images.githubusercontent.com/13720708/121763161-b37bc080-cb31-11eb-8b90-3d9a15ca460c.png)

**Screenshots or screencast URL:**
![image](https://user-images.githubusercontent.com/13720708/121763171-d8703380-cb31-11eb-8be5-592c705fe8e5.png)

**Bug Type:**
UX / Usability Bug

Device/OS/Browser Information:
Windows 10; Microsoft Edge, Opera

**Comment by nithyaasworld** [comment]
Good catch @Chapre Please keep checking topcoder challenge page for any active Bug hunt challenges and participate. Also, be part of dev-challenges channel of Topcoder Community Slack workspace to get immediate notification about all the new challenges.
